examples/pennylane_run_barren_plateaus.py

```diff
@@ -0,0 +1,198 @@
+r"""
+.. _barren_plateaus:
+
+Barren plateaus in quantum neural networks
+==========================================
+*Author: Shahnawaz Ahmed (shahnawaz.ahmed95@gmail.com)*
+
+In classical optimization, it is suggested "that saddle
+points, not local minima, provide a fundamental impediment
+to rapid high dimensional non-convex optimization".
+(Dauphin et. al, 2014).
+
+The problem of such barren plateaus manifests in a different
+form in variational quantum circuits which are at the heart
+of techniques such as quantum neural networks or approximate
+optimization eg., QAOA (Quantum Adiabatic Optimization Algorithm).
+
+While starting from a parameterized
+random quantum circuit seems like a good unbiased choice if
+we do not know the problem structure, McClean et. al., 2018,
+show that
+
+*"for a wide class of reasonable parameterized quantum
+circuits, the probability that the gradient along any
+reasonable direction is non-zero to some fixed precision
+is exponentially small as a function of the number
+of qubits."*
+
+Thus, random quantum circuits might not be the best
+option to choose while implementing variational quantum
+algorithms.
+
+
+.. figure:: ../../examples/figures/barren_plateaus/surface.png
```

Comment:
maybe I'm imaginging things, but there's an extra yellow "swipe" on top of the measurement block